core/model/modx/processors/resource/update.class.php

```diff
@@ -452,6 +452,18 @@ public function checkForUnPublishOnSiteStart() {
      * @return boolean
      */
     public function checkDeletedStatus() {
+	    if (1 == $this->getProperty('deleted', null)) {
+	    	if ($this->modx->getOption('site_start') == $this->getProperty('id')) {
```

Comment:
Mhhh thats a good point, didn't think about that. But then there is a bug in the resource/delete processor because that contains the same code:

if ($this->modx->getOption('site_start') == $this->resource->get('id')) {
    return $this->failure($this->modx->lexicon('resource_err_delete_sitestart'));
}
if ($this->modx->getOption('site_unavailable_page') == $this->resource->get('id')) {
    return $this->failure($this->modx->lexicon('resource_err_delete_siteunavailable'));
}